src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1313,6 +1313,11 @@ theorem getElem_shiftLeft' {x : BitVec w₁} {y : BitVec w₂} {i : Nat} (h : i
     (x <<< y)[i] = (!decide (i < y.toNat) && x.getLsbD (i - y.toNat)) := by
   simp
 
+theorem shiftLeft_eq_zero (x : BitVec w) (n : Nat) (hn : w ≤ n) : x <<< n = 0#w := by
+  ext i hi
+  simp [hn, hi]
+  omega
```

Comment:
```suggestion
theorem shiftLeft_eq_zero {x : BitVec w} {n : Nat} (hn : w ≤ n) : x <<< n = 0#w := by
  ext i hi
  simp [hn, hi]
  omega
```

Also, should this be tagged `@[simp]`?

Reply:
I've been a bit conservative in tagging theorems `@[simp]`, I want to take time and sweep through everything we have. But I think this is clearly a `simp` lemma.